modules/core/src/main/scala/org/tessellation/config/types.scala

```diff
@@ -61,17 +62,26 @@ object types {
     remainingWeight: Weight
   )
 
+  case class OneTimeReward(epoch: EpochProgress, address: Address, txnAmount: TransactionAmount)
+
   case class RewardsConfig(
     programs: EpochProgress => ProgramsDistributionConfig = mainnetProgramsDistributionConfig,
-    rewardsPerEpoch: SortedMap[EpochProgress, Amount] = mainnetRewardsPerEpoch
+    rewardsPerEpoch: SortedMap[EpochProgress, Amount] = mainnetRewardsPerEpoch,
+    oneTimeRewards: List[OneTimeReward] = List(
+      // Transferring current balance of 43,320,500 from DAGSTARDUSTCOLLECTIVEHZOIPHXZUBFGNXWJETZVSPAPAHMLXS
+      //  adjusted by 1e8 to build correct transaction amount.
+      OneTimeReward(EpochProgress(1358392L), stardustNewPrimary, TransactionAmount(4_332_050_000_000_000L))
+    )
```

Comment:
have you calculated the balance for that epoch progress, or is it a current balance?
IMO, you should calculate how much rewards will be created till that epoch progress, and transfer the exact amount. otherwise, it will cause a drop/increase in the total supply. cc: @ryle-goehausen 
(or: see my other comment how to take an exact balance during snapshot creation process)

Reply:
David just proposed doing this as an additional delta in the future where we transfer any remaining amount as a secondary one time reward, which seems fine to me vs. estimating exactly. 

We took the exact current balance as of yesterday rounded to a given value (specifically so we can verify and embed it into the code so anyone else can very quickly verify it's correct.)

The estimated amount between now and 3 days from now is very small. This will cause a minor drop in supply (3 days of the 5% rewards) which is fine, but we're going to make a second operation later to track the exact amount delta / transfer the remainder. This way we'll keep track of it in the code to make any external verification much easier (as someone will be able to check the original balance from the explorer, and the new balance.)

Reply:
See other comment, we're splitting into 2 releases to make it exact